zerver/lib/users.py

```diff
@@ -483,6 +486,7 @@ def format_user_row(
             client_gravatar=client_gravatar,
         )
 
+    result["delivery_email"] = None
```

Comment:
I think it might be cleaner if this were an `else` block for the next `if` statement -- that way, this field only has a single value within this function, rather than having one set and then overwritten.

Reply:
Updated.